pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -20,36 +21,42 @@ def load_earth_free_air_anomaly(
     ] = "01d",
     region: Sequence[float] | str | None = None,
     registration: Literal["gridline", "pixel", None] = None,
+    data_source: Literal["faa", "faaerror"] = "faa",
```

Comment:
I don't like the parameter name `data_source` here, since these two datasets acutally comes from the same source.

Maybe `uncertainty=True` to use the `faaerror` data?